website/docs/getting-started/setup/instance-configuration/authentication/google-login.md

```diff
@@ -7,64 +7,63 @@ sidebar_position: 1
 
 # Google OAuth
 
-Google OAuth is one of the most widely used custom authentication integrations. Appsmith allows you to integrate with Google for authenticating users to your apps.
+Appsmith allows you to integrate with Google to authenticate users to your apps.
 
-![Click to expand](</img/Google_Login.png>)
+## Configure Google Cloud
 
-To enable Google Sign in, login to your [google cloud console](https://console.cloud.google.com/)
+To enable Google Sign in, log in to your [google cloud console](https://console.cloud.google.com/) and follow the steps below:
 
-**1. Navigate to the** [**OAuth consent section**](https://console.cloud.google.com/apis/credentials/consent) **under APIs & Services**
+1. Navigate to the [**OAuth consent screen**](https://console.cloud.google.com/apis/credentials/consent) under **APIs & Services**.
 
-![Click to expand](</img/Google_OAuth_Consent_1.png>)
+  ![Click to expand](</img/Google_OAuth_Consent_1.png>)
 
-**2. Configure the consent screen with the domain on which you want to host Appsmith**
+2. Configure the consent screen with the domain you want to host Appsmith.
 
-![Click to expand](</img/Google_OAuth_Consent.png>)
+  ![Click to expand](</img/Google_OAuth_Consent.png>)
 
-**3. Navigate to the** [**credentials section**](https://console.cloud.google.com/apis/credentials) **and create new OAuth Credentials**
+3. Navigate to the [**credentials**](https://console.cloud.google.com/apis/credentials) and create new OAuth Credentials.
 
-![Click to expand](</img/Google_OAuth_Creds.png>)
+  ![Click to expand](</img/Google_OAuth_Creds.png>)
 
-**4. Configure the OAuth Credentials for a web application**
+4. Select the application type as a Web application. Update the following fields:
 
-**Javascript Origins:** the domain on which you are hosting Appsmith with HTTPS **`(ex: https://app.appsmith.com)`**
+    1. **Javascript Origins:** the domain on which you are hosting Appsmith with HTTPS **`(ex: https://app.appsmith.com)`**
+    2. **Redirect URIs:** append **/login/oauth2/code/google** to your Javascript origins **`(ex: https://app.appsmith.com/login/oauth2/code/google)`**
 
-**Redirect URIs:** append **/login/oauth2/code/google** to your Javascript origins **`(ex: https://app.appsmith.com/login/oauth2/code/google)`**
+5. Click Create. You can get the Client ID and Client secret under "OAuth 2.0 Client IDs" in the Credentials tab.
 
-![Click to expand](</img/Google_Oauth_Creds_2.png>)
+  ![Click to expand](</img/Google_Oauth_Creds_2.png>)
 
-**5. Update the**[ **instance configuration**](../) **with the following fields**:
+## Configure your instance  
 
-  * **Configure via UI (recommended)**:
+Update the [**instance configuration**](../) with the following fields:
```

Comment:
There are two ways to configure the self-hosted Appsmith instance: Admin Settings(link to section below)
and Environment Variables(link to section below)